C/common/include/utils.h

```diff
@@ -45,6 +45,16 @@ static const string getDataDir()
 	return (dataDir ? string(dataDir) : string(getRootDir() + "/data"));
 }
 
+/**
+ * @brief Constructs the path for the debug-trace dir based on the directory returned by getDataDir().
```

Comment:
I wouldn't document a subroutine based on the behavior of another. How about "Constructs the path for the debug-trace subdirectory in the Fledge data directory" ?